### ISSUE TITLE
Replace CHANGES.txt with CHANGES.md (at least file name)

### ISSUE THREAD

**Issue by asergeev-sc** [OpusCapitaBES/js-npm-scripts#7]

**Comment by stamkov-sc** [comment]
CHANGES.txt => CHANGELOG.md

**Comment by kvolkovich-sc** [comment]
Done.

**Comment by kvolkovich-sc** [comment]
Output format has been changed:

* Added links to commits diff
* Added short git commit hash  

 ### From:

```

Release 0.0.1-beta.15 Fri May 26 2017 11:41:00 GMT+0300 (+03)
=======================================================

 - Change peerDependencies (Kirill Volkovich <volkovich@scand.com>, Fri May 26 11:40:27 2017 +0300)
 - Updating to a version to  0.0.1-beta.15 (Kirill Volkovich <volkovich@scand.com>, Mon May 22 16:42:46 2017 +0300)
 - Fix lint (Kirill Volkovich <volkovich@scand.com>, Mon May 22 16:42:06 2017 +0300)
 - Add translations to i18nLinker (Kirill Volkovich <volkovich@scand.com>, Mon May 22 16:40:44 2017 +0300)

Release 0.0.1-beta.14 Mon May 22 2017 16:42:34 GMT+0300 (+03)
=======================================================

 - Fix lint (Kirill Volkovich <volkovich@scand.com>, Mon May 22 16:42:06 2017 +0300)
 - Add translations to i18nLinker (Kirill Volkovich <volkovich@scand.com>, Mon May 22 16:40:44 2017 +0300)
```

### To:

```
## [v0.3.0-beta.1](https://github.com/OpusCapita/react-dates/compare/v0.3.0-beta.0...v0.3.0-beta.1) (Wed, 05 Jul 2017 10:23:57 GMT)
 - Fix dash centering styles of DateRangeInput (Kirill Volkovich <volkovich@scand.com>, ed08b24)

## [v0.3.0-beta.0](https://github.com/OpusCapita/react-dates/compare/v0.2.16...v0.3.0-beta.0) (Wed, 05 Jul 2017 09:44:04 GMT)
 - Update minor package version (Kirill Volkovich <volkovich@scand.com>, 29a36e2)
 - Merge pull request #18 from OpusCapita/feature/17
    
    Date range: impossible to enter value by keyboard, fix 'de' l11n (56f6d3f 5f0c236, c8ad4e2)
```

**Comment by kvolkovich-sc** [comment]
Review needed.
